packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol

```diff
@@ -890,17 +909,19 @@ library NounsDAOV3Proposals {
 
     function createNewProposal(
         NounsDAOStorageV3.StorageV3 storage ds,
-        uint256 proposalId,
+        uint32 proposalId,
         uint256 proposalThreshold_,
         uint256 adjustedTotalSupply,
-        ProposalTxs memory txs
+        ProposalTxs memory txs,
+        uint32 clientId
     ) internal returns (NounsDAOStorageV3.Proposal storage newProposal) {
         uint64 updatePeriodEndBlock = SafeCast.toUint64(block.number + ds.proposalUpdatablePeriodInBlocks);
         uint256 startBlock = updatePeriodEndBlock + ds.votingDelay;
         uint256 endBlock = startBlock + ds.votingPeriod;
 
         newProposal = ds._proposals[proposalId];
         newProposal.id = proposalId;
+        newProposal.clientId = clientId;
```

Comment:
similar to the comment on cast vote, wouldn't it be cool to include the client ID in new proposal events? let's discuss

Reply:
agree